gcovr/tests/use-existing/main.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+
+
+int foo(int param) {
+  if (param) {
+     return 1; //std::cout << "param not null." << std::endl;
+  } else {
+     return 0; //std::cout << "param is null." << std::endl;
+  }
+}
+
+
+int main(int argc, char* argv[]) {
+  foo(
+     0
+     )
+     ;
```

Comment:
This fancy code layout is not relevant to the gcovr behaviour being tested: the `-v` option.

Reply:
c.f. updated README file => same cpp file as simple1 test